### PR TITLE
Infiltration Kit & Overpressure Protection

### PR DESCRIPTION
**Pull request by Geevies**
* Added an infiltration kit to the Devices and Tools uplink. It spawns a case containing a mask, a uniform, gloves, and shoes, capable of protecting you from the vacuum of space, as well as two atmospheres of overpressure.
* Ported max and min pressure tolerances for items and clothing, meaning that certain gear will now protect better from overpressure, such as atmos voidsuits, firesuits, bomb suits, and to a lesser extent, engineering voidsuits.

image

Infiltrator sprites ported (and tweaked) from: https://github.com/tgstation/tgstation/pull/49608
Pressure changes ported (and tweaked) from: https://github.com/Baystation12/Baystation12/pull/25576

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/modules/clothing/sets/infiltrator.dm [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+	name = "infiltration balaclava"
+	desc = "A close-fitting airtight balaclava that can be connected to an air supply."
+	icon = 'icons/clothing/kit/infiltrator.dmi'
+	icon_state = "mask"
+	item_state = "mask"
+	icon_auto_adapt = TRUE
+	icon_supported_species_tags = list("una", "taj")
+	contained_sprite = TRUE
+	adjustable = FALSE
+	item_flags = THICKMATERIAL | INJECTIONPORT | AIRTIGHT
```
Comment (fernerr) [inline code/modules/clothing/sets/infiltrator.dm:11]:
```suggestion
	item_flags = THICKMATERIAL | INJECTIONPORT | AIRTIGHT
	flags_inv = HIDEFACE|BLOCKHAIR
```

Comment (fernerr) [reply code/modules/clothing/sets/infiltrator.dm:11]: Else their identity will remain unhidden.

**Review by fernerr — CHANGES_REQUESTED** [review]

File: code/modules/clothing/sets/infiltrator.dm (hunk 1 above)
Comment (Geevies) [reply code/modules/clothing/sets/infiltrator.dm:11]: good idea

**Review by fernerr — APPROVED** [review]

**Review by JohnWildkins — APPROVED** [review]

**Pull request merged**